### ISSUE TITLE
Support different base colours

### ISSUE THREAD

**Issue by edran** [nashamri/spacemacs-theme#61]
First off, let me give you a huge amount of thanks for making such a nice theme. Making Emacs pretty is always quite a challenge, and you have managed pretty well so far :)

I was just wondering if it'd be possible to have different versions of the dark theme where the base colour is not just purple (e.g. having a `spacemacs-red`, `spacemacs-blue`, etc.), and whether you think this might be feasible to achieve with the current scheme (e.g. by for instance adding some `$COLOR` to the current ones).

**Comment by nashamri** [comment]
Hey @edran thanks for the kind words :smile: 

You can do that right now with spacemacs-theme. For example, just add this to your `dotspacemacs/user-init`:

```
  (custom-set-variables '(spacemacs-theme-custom-colors
                          '((act1 . "#ff0000")
                            (act2 . "#0000ff")
                            (base . "#ffffff"))))
```
To change the `act1`, `act2` and `base` .

**Comment by edran** [comment]
Excellent :) 

The only issue with this particular solution is that it's harder to deal with both `dark` and `light` theme, so I wonder if it'd be easier to ship the theme directly with a custom function?

**Comment by nashamri** [comment]
Yes the solution I mentioned won't work for both variants. I need to play with this a bit to see what's the best way to support both variants.
